src/node_crypto.cc

```diff
@@ -4878,6 +4878,27 @@ void GetHashes(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void GetCurves(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Local<Array> arr = Array::New(env->isolate());
+  size_t len = EC_get_builtin_curves(NULL, 0);
+  EC_builtin_curve* curves =
+    reinterpret_cast<EC_builtin_curve*>(malloc(sizeof(EC_builtin_curve) * len));
```

Comment:
Style: four spaces indent for line continuations and can you use `sizeof(*curves)`?